src/Server.ts

```diff
@@ -3,6 +3,7 @@ import * as bodyParser from 'body-parser';
 import Iconfig from './config/iconfig';
 import { errorHandler, notFoundRoute } from './libs/routes';
 import router from './router';
+import Database from './libs/database';
```

Comment:
database folder name must be starting from capital

Reply:
Done